src/providers/ModalProvider/modal-reducer.js

```diff
@@ -6,6 +6,7 @@ export const actions = {
 export const initialState = {
   show: false,
   header: {
+    withHeader: false,
```

Comment:
1. `withHeader` should be outside the `header` object.
2. A large majority of the modals are `withHeader=true` so why not place the default to `true`?